### PR TITLE
add PermissionScheme Resource

### PR DESCRIPTION
**Pull request by Steffen223**
make PermissionScheme Resource also available when using pycontribs/jira.
tox lint passed. But I had issues with my environment using tox tests. Using this branch in a our script worked successful

### CODE REVIEW

File: jira/client.py [hunk 1]
```diff
@@ -2019,6 +2020,15 @@ def votes(self, issue: str) -> Votes:
         """
         return self._find_for_resource(Votes, issue)
 
+    @translate_resource_args
+    def project_permissionscheme(self, project):
+        """Get a PermissionScheme Resource from the server.
+
+        :param project: ID or key of the project to get the permissionscheme for
+        :rtype: PermissionScheme
+        """
```
Comment (adehad) [inline jira/client.py:2029]: Let's use the Google style docstring 

```suggestion

        Args:
            project (str): ID or key of the project to get the permissionscheme for

        Returns:
        PermissionScheme: The permission scheme
        """
```

File: jira/client.py [hunk 2]
```diff
@@ -2019,6 +2020,15 @@ def votes(self, issue: str) -> Votes:
         """
         return self._find_for_resource(Votes, issue)
 
+    @translate_resource_args
+    def project_permissionscheme(self, project):
```
Comment (adehad) [inline jira/client.py:2024]:
```suggestion
    def project_permissionscheme(self, project: str):
```

File: jira/resources.py [hunk 3]
```diff
@@ -798,6 +799,17 @@ def __init__(
...
+class PermissionScheme(Resource):
+    """Permissionscheme information on an project."""
+
+    def __init__(self, options, session, raw=None):
+        Resource.__init__(
+            self, "project/{0}/permissionscheme?expand=user", options, session
+        )
+        if raw:
+            self._parse_raw(raw)
+
```
Comment (adehad) [inline jira/resources.py:811]:
```suggestion
        self.raw: Dict[str, Any] = cast(Dict[str, Any], self.raw)
```

**Review by adehad — CHANGES_REQUESTED** [review]
looking good ! Tests can be a bit tricky as you need to setup docker, but if you can try something locally feel free to add a test file to the `test/resources/` folder and we can see how it runs on the CI

**Comment by Steffen223** [comment]
Hello @adehad , a test has been added to https://github.com/Steffen223/jira/blob/master/tests/resources/test_project.py#L204

**Comment by adehad** [comment]
Test failure is unrelated to this, seems like we need to reconfigure something on our github actions end. Will try and sort this out and then re-run the jobs.

**Comment by adehad** [comment]
Looks like a few linting error popped up could you run `tox -e lint` ?
Can you also first rebasing / merge master back in? As we just merged in an update to our lint

**Comment by Steffen223** [comment]
Hello @adehad , I linted, updated and merged from master.

**Review by adehad — APPROVED** [review]

**Comment by adehad** [comment]
Awesome, thank you for your contribution !

**Pull request merged**